tests/frontend/specs/select_formatting_buttons.js

```diff
@@ -102,8 +107,8 @@ describe('select formatting buttons when selection has style applied', function
         applyStyleOnLineAndSelectIt(FIRST_LINE, style, done);
       });
 
-      after(function () {
-        undo();
+      after(async function () {
+        await undo();
```

Comment:
because the undo function is not async, does this work?

Reply:
Didn't I make undo async?